fred2/orienteditor.cpp

```diff
@@ -182,9 +182,27 @@ BOOL orient_editor::OnInitDialog()
 	return TRUE;
 }
 
-bool orient_editor::close(float val, const CString &str)
+bool orient_editor::close(float val, const CString &input_str)
 {
-	float diff = val - convert(str);
+	CString str;
+	str.Format("%.01f", val);
+	val = convert(str);
+
+	float input_val = convert(input_str);
+
+	float diff = val - input_val;
+	return diff > -input_threshold && diff < input_threshold;
+}
+
+bool orient_editor::angle_close(float rad, const CString &input_str)
+{
+	CString str;
+	str.Format("%.01f", to_degrees(rad));
```

Comment:
Same comments as above that an explanation here would be nice and also if it's a concern that `input_rad` doesn't seem to get formatted this way.

Reply:
See upcoming commit.